ignition/modules/OGFoundersModule.ts

```diff
@@ -8,13 +8,20 @@ export const OGFoundersModule = buildModule('OGFounders', m => {
   const deployer = m.getAccount(0)
   const stRIFAddress = m.getParameter('stRIFAddress')
   const firstProposalDate = m.getParameter('firstProposalDate')
+  const maxSupply = m.getParameter('maxSupply')
+  const ipfsFolderCid = m.getParameter('ipfsFolderCid')
 
   // deploy proxy
   const proxy = m.contract('ERC1967Proxy', [
     implementation,
-    m.encodeFunctionCall(implementation, 'initialize', [deployer, stRIFAddress, firstProposalDate], {
-      id: 'Proxy',
-    }),
+    m.encodeFunctionCall(
+      implementation,
+      'initialize',
+      [deployer, stRIFAddress, firstProposalDate, maxSupply, ipfsFolderCid],
```

Comment:
Why are you setting maxSupply from the deployment? What's the purpose of this change?

Reply:
Can you guarantee that tomorrow Toby wont come and ask to replace some images or to add some? It already happened in the past and will happen again. So it must be possible to change te number of tokens in the contract